public/stylesheets/app/_overrides.scss

```diff
@@ -5,3 +5,7 @@
 .moj-sub-navigation__link {
   @extend .govuk-link--no-visited-state;
 }
+
+.moj-pagination__results {
+  @extend .govuk-body;
```

Comment:
This is broken in the Pagination component so this override is necessary for now.